SillyBusinessInc/SillyBusinessGame
# Feat/#119 crumbs keeping track

## Purpose of this PR:
We want to save the modifiers and mulitpliers, crumbs and health. we did this by using the savesystem fabio made

## How to test:
In Si Wai > StatsChangeScene you can test the saving.
To get the stats, use `GlobalReference.PermanentPlayerStatistic.Get<type>(key)`, like `GlobalReference.PermanentPlayerStatistic.Get<float>("health")` to get the saved value

when enemy attacks the player, it should display in the console the old health and new health. To get the crumbs, go to the Player in hierarchy. In the inspector, find Player (script) and click on the 3 dots, at the bottom it says 'add 30 crumbs' and it will automatically add 30 crumbs. stop the game and play it again and you should see the updated amount of crumbs

### links: 
[Ticket](https://github.com/orgs/SillyBusinessInc/projects/1?pane=issue&itemId=87872213&issue=SillyBusinessInc%7CSillyBusinessGame%7C119)
